src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -275,7 +280,7 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   _handleKeydown(event: KeyboardEvent): void {
     if (event.keyCode === ESCAPE && this.panelOpen) {
       this._resetActiveItem();
-      this.closePanel();
+      this._escapeEventStream.next(event);
```

Comment:
Since it only fires for escape presses, I don't think it's necessary to pass along the event.